docs/03-exports/aliases-api.include

```diff
@@ -991,6 +991,7 @@
 .. |ITypeObjectRegistry-api| replace:: :cpp:class:`ITypeObjectRegistry <eprosima::fastdds::dds::xtypes::ITypeObjectRegistry>`
 .. |ITypeObjectRegistry::get_type_object| replace:: :cpp:func:`ITypeObjectRegistry::get_type_object <eprosima::fastdds::dds::xtypes::ITypeObjectRegistry::get_type_object>`
 .. |TypeObjectUtils-api| replace:: :cpp:class:`TypeObjectUtils <eprosima::fastdds::dds::xtypes::TypeObjectUtils>`
+.. |XTypesUtils-idl_serialize-api| replace:: :cpp:func:`idl_serialize <eprosima::fastdds::dds::idl_serialize>`
```

Comment:
This is not enough for referencing the function, you have to add it to the API section. 

See how it's done for the [JSON serializer](https://github.com/eProsima/Fast-DDS-docs/blob/144d5224418e1dbe51de2262e6e6aec73d2d520a/docs/fastdds/api_reference/dds_pim/xtypes/utils.rst?plain=1).